metrics.go

```diff
@@ -66,6 +98,13 @@ func (c *ClientIMPL) PerformanceMetricsByVolume(ctx context.Context, entityID st
 	return resp, err
 }
 
+// VolumeMirrorTransferRate - Volume Mirror Transfer Rate
+func (c *ClientIMPL) VolumeMirrorTransferRate(ctx context.Context, entityID string) ([]VolumeMirrorTransferRateResponse, error) {
+	var resp []VolumeMirrorTransferRateResponse
+	err := c.mirrorTransferRate(ctx, &resp, entityID, 2000)
```

Comment:
Can this `2000` be a constant value instead of hardcoding it here like this way?

Reply:
done